app/javascript/admin/controllers/config_controller.js

```diff
@@ -402,7 +402,7 @@ export default class ConfigController extends Controller {
 
   activateMissingKeysModal(providers) {
     this.configModalAnchorTarget.innerHTML = adminModal({
-      title: 'Setup not complete',
+      title: 'Getting started',
```

Comment:
Everything seems to work as expected with this change, though I'm not 100% sure if this is a solid change.